1223-curator_worklist/1375-curator_views_doi_request_tab_part_ii.feature

```diff
@@ -0,0 +1,12 @@
+    @1375
+    Scenario: Curator views DOI Request tab (Part II)
+        Given that the user opens My Worklist
```

Comment:
Each scenario should be able to stand by itself (combined with a possibly existing background).
The user is a Curator, no? Or is it any user (Anonymous, Creator,Curator, etc.) that will be able to see a possibly empty list?